### ISSUE TITLE
Android Tabs: crash when switching css theme and navigate back

### ISSUE THREAD

**Issue by felixkrautschuk** [nativescript-community/ui-material-components#315]
### If the demo apps cannot help and there is no issue for your problem, tell us about it

When switching the css theme and then navigating back to a page that contains the MDTabs component, the App crashes on Android: `java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.View android.widget.LinearLayout.getChildAt(int)' on a null object reference`

I define the styles for Tabs like this:

theme-light.css
```
MDTabStrip {
    selected-item-color: blue;
    un-selected-item-color: gray;
    ripple-color: lightblue;
    highlight-color: orange;
}

MDTabStripItem {
    background-color: #f1f1f1;
}

MDTabContentItem {
    background-color: white;
}
```
(same structure with other colors for theme-dark.css)

And I switch the theme like this:

```
import {Frame, fromObject, knownFolders} from "@nativescript/core";
import {setCssFileName} from "@nativescript/core/application";

setCssFileName(knownFolders.currentApp().path + "/theme-dark.css");
Frame.topmost().currentPage._onCssStateChange();
```

I use this way of switching between 50 themes in our real app now for a very long time now, it is working with all {N} components so far, also with the old TabView core-component, so I don't think there is a gernal problem with that approach.

https://user-images.githubusercontent.com/6443021/125062005-37cc4f80-e0ae-11eb-9cc6-63c1c0835f6f.mov

**My observation:**
The issue does NOT occur when removing the following style:

```
MDTabContentItem {
    background-color: white;
}
```

BUT I need it, otherwise the TabStrip background-color is not getting updated on Android, when you switch the theme while you have a page with Tabs open (which can happen in our real app).

Crash log:

> System.err: An uncaught Exception occurred on "main" thread.
> System.err: Calling js method onCreateView failed
> System.err: Error: java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.View android.widget.LinearLayout.getChildAt(int)' on a null object reference
> System.err: 
> System.err: StackTrace:
> System.err: updateAndroidItemAt(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-tabs/tabs.android.js:617:0)
> System.err:     at updateItem(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-tabs/tabs.android.js:652:0)
> System.err:     at setTabBarItemBackgroundColor(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-tabs/tabs.android.js:658:0)
> System.err:     at [backgroundColor:setNative](file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-core/tab-navigation-base/tab-strip-item/index.js:184:0)
> System.err:     at applyPendingNativeSetters(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/properties/index.js:1118:0)
> System.err:     at initNativeView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/properties/index.js:1077:0)
> System.err:     at onResumeNativeUpdates(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:737:22)
> System.err:     at _resumeNativeUpdates(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:280:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:235:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view/index.android.js:291:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-core/tab-navigation-base/tab-strip-item/index.js:56:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at callFunctionWithSuper(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:312:0)
> System.err:     at callLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at loadView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:458:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:237:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-core/tab-navigation-base/tab-strip/index.js:30:0)
> System.err:     at eachChild(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-core/tab-navigation-base/tab-strip/index.js:29:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:236:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view/index.android.js:291:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at callFunctionWithSuper(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:312:0)
> System.err:     at callLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at loadView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:458:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:237:0)
> System.err:     at eachChild(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-core/tab-navigation-base/tab-navigation-base/index.js:39:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:236:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view/index.android.js:291:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript-community/ui-material-tabs/tabs.android.js:440:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at callFunctionWithSuper(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:312:0)
> System.err:     at callLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at loadView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:458:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:237:0)
> System.err:     at eachChildView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/content-view/index.js:65:0)
> System.err:     at eachChildView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/page/page-common.js:106:0)
> System.err:     at eachChild(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:236:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view/index.android.js:291:0)
> System.err:     at onLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/page/index.android.js:38:0)
> System.err:     at (file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at callFunctionWithSuper(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:312:0)
> System.err:     at callLoaded(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
> System.err:     at onCreateView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/frame/index.android.js:742:0)
> System.err:     at onCreateView(file: app/webpack:/@nativescript/template-hello-world-ts/node_modules/@nativescript/core/ui/frame/fragment.android.js:25:0)
> System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
> System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
> System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
> System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
> System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
> System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
> System.err:     at com.tns.FragmentClass.onCreateView(FragmentClass.java:55)
> System.err:     at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2699)
> System.err:     at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:320)
> System.err:     at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1199)
> System.err:     at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1368)
> System.err:     at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1446)
> System.err:     at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1509)
> System.err:     at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
> System.err:     at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2181)
> System.err:     at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2004)
> System.err:     at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1959)
> System.err:     at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1861)
> System.err:     at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:413)
> System.err:     at android.os.Handler.handleCallback(Handler.java:883)
> System.err:     at android.os.Handler.dispatchMessage(Handler.java:100)
> System.err:     at android.os.Looper.loop(Looper.java:214)
> System.err:     at android.app.ActivityThread.main(ActivityThread.java:7356)
> System.err:     at java.lang.reflect.Method.invoke(Native Method)
> System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
> System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
> System.err: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.View android.widget.LinearLayout.getChildAt(int)' on a null object reference
> System.err:     at com.nativescript.material.core.TabsBar.updateItemAt(TabsBar.java:195)
> System.err:     ... 26 more

### Which platform(s) does your issue occur on?
- Android
- tested on 5, 7, 10, 11
- emulator andr device

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.0.2
- Cross-platform modules: 8.0.8
- Runtime(s): 8.0.0
- Plugin(s): ui-material-tabs 5.3.9 (also tested older versions up to 5.2.0, same crash)

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

Folow steps from gif
- navigate forwards to second-page (ActionItem)
- switch the theme to dark
- navigate back -> crash

### Is there any code involved? 
[ns-tabs-theme-issue-android.zip](https://github.com/nativescript-community/ui-material-components/files/6790591/ns-tabs-theme-issue-android.zip)

**Comment by farfromrefug** [comment]
@felixkrautschuk fixed in 5.3.10.

**Comment by felixkrautschuk** [comment]
@farfromrefug thanks for the fast fix!
